core/utils/utils.go

```diff
@@ -1113,3 +1113,13 @@ func MinKey[U any, T constraints.Ordered](elems []U, key func(U) T) T {
 
 	return min
 }
+
+func Min[T constraints.Ordered](firstElem T, elems ...T) T {
```

Comment:
This is already implemented in `mathutil` (https://github.com/smartcontractkit/chainlink/blob/f61508b02ff369db2baafbfe303ffd2ec709b587/core/utils/mathutil/mathutil.go#L15)